### ISSUE TITLE
Search function finds pages that don't contain the search text.

### ISSUE THREAD

**Issue by Docmale** [digitalfabrik/integreat-app#2741]
### Describe the Bug

The search with strings of up to 4 characters finds pages in which the search text does not appear. I have found several examples of 3 and 4 long strings, but found no example for longer strings.

### Steps to Reproduce

1. Go to 'Landeshauptstadt München' (Deutsch)
2. Enter search string 'asdf'
3. Find page 'Arbeitsunfall'
4. more:
5. Enter search string 'DRK'
6. Find page 'Alten- und Service-Zentren' and other

### Expected Behavior

Search function should not display pages that does not contain the serch string

### Additional Information

![Search](https://github.com/digitalfabrik/integreat-app/assets/21236324/1da37d77-b0e9-4940-944c-900c1bfa7f4e)

**Comment by osmers** [comment]
https://chat.tuerantuer.org/digitalfabrik/pl/jrpcm3ar8tbh7kc3zawo5zsrxh
There was a discussion about this already - it has to do with the fuzzy search properties.

**Comment by steffenkleinle** [comment]
Thanks for investigating @lunars97, I am fine with closing this. @LeandraH would you want to investigate this a little more as you have worked with minisearch before?
Btw, the default fuzzy value is `0.2`. https://github.com/lucaong/minisearch/blob/d46245015f34932058861ebeb1eb7fdf97ebaaae/src/MiniSearch.ts#L1713

**Comment by LeandraH** [comment]
No, I fully agree with @lunars97 , it's a better UX to have some additional unexpected results than to have missing expected results.
